scripts/fvt/db_test.sh

```diff
@@ -127,7 +127,7 @@ EOF
    cp $MYSQLSERVERCONFIG $PGSQLSERVERCONFIG
    cp $MYSQLSERVERCONFIG2 $PGSQLSERVERCONFIG2
    sed -i "s/type: mysql/type: postgres/
-          s/datasource:.*/datasource: host=localhost port=$POSTGRES_PORT user=postgres password=postgres dbname=fabric_ca/" \
+          s/datasource:.*/datasource: host=postgres port=$POSTGRES_PORT user=postgres password=postgres dbname=fabric_ca sslmode=disable/" \
    $PGSQLSERVERCONFIG $PGSQLSERVERCONFIG2
 }
```

Comment:
You're just copying this but we could just cat the source through sed to create the target file. That gets rid of the `-i` flag and it's inconsistent behavior depending on the version of `sed` that's being used.